### DISCUSSION TITLE
帮帮忙！！！

### DISCUSSION THREAD

**Discussion by 57480uiop** [share121/inter-knot#3344] [其他]
网页加载不出来！获取文章信息失败

**Comment by sky-zht** [comment]
可能是网的问题？ Github 是国外网站，所以访问又慢又不稳定

**Comment by PDWYX** [comment]
up发了新的视频， BV1MPWAe3E23，里面有解决办法，清空缓存<img src="https://bbs-static.miyoushe.com/static/2024/03/21/dab9bdd5012200a2902c8ae19dc1c8c9_4053703670259707891.png" alt="丽都漫步-可琳" width="75" >
